mxcube3/ui/components/Tasks/warning.js

```diff
@@ -0,0 +1,31 @@
+const warn = (values, props) => {
```

Comment:
Why are you doing this, is it because you want to tell the users that the value differs or is it because the validation somehow is to strict ?

Reply:
simply to warn the users. Some mistakes were made here by users because it was not clear. This is only a warning, visual thing.
The strict validation is still needed.

Reply:
Okey, I need to discuss this one with Daniele, Ill get back to you later